src/test/denialofservice_tests.cpp

```diff
@@ -197,7 +197,7 @@ BOOST_AUTO_TEST_CASE(stale_tip_peer_management)
 
     // Update the last announced block time for the last
     // peer, and check that the next newest node gets evicted.
-    UpdateLastBlockAnnounceTime(vNodes.back()->GetId(), GetTime());
+    peerLogic->UpdateLastBlockAnnounceTime(vNodes.back()->GetId(), GetTime());
```

Comment:
You can remove the `UpdateLastBlockAnnounceTime()` declaration on line 39 (it's now no longer used).

Reply:
Done, good catch!